.github/workflows/dashboard.yml

```diff
@@ -0,0 +1,20 @@
+name: Dashboard CI/CD
+on:
+  push:
+    branches:
```

Comment:
I wonder if we should only run this with a release? It seems a bit weird that the dashboard would update ahead of the actual deployment :).